src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,118 @@
+package duke.ui;
+
+import duke.task.Task;
+
+/**
+ * Ui class that handles showing the user texts.
+ *
+ * @author Elgin
+ */
+public class Ui {
+    /**
+     * Bye to user (End process message).
+     *
+     * @return String representation of Bye message.
+     */
+    public String getByeMsg() {
+        return "Bye. Hope to see you again soon!\n" + "Application closing in 3 seconds...";
+    }
+
+    /**
+     * Formats a task added message for Duke.
+     *
+     * @return Message conveying that task has successfully been added, and the current number of tasks.
+     */
+    public String getTaskAddedMsg(int newTaskCount, Task task) {
+        return "Got it. I've added this task:\n" + "  " + task + "\n" + "Now you have "
+                + newTaskCount + " tasks in the list.";
+    }
+
+    /**
+     * Formats a task deleted message for Duke.
+     *
+     * @return Message conveying that user that has successfully deleted a task, and the number of tasks left
+     */
+    public String taskDeletedMsg(int newTaskCount, String taskDescription) {
+        return "Noted. I've removed this task:\n  " + taskDescription + "\n" + "Now you have "
+                + newTaskCount + " tasks in the list";
+    }
+
+    /**
+     * Formats a message signaling a task has been successfully marked.
+     *
+     * @param taskDescription The description of the task marked.
+     */
+    public String getTaskMarkedMsg(String taskDescription) {
+        return "Nice! I've marked this task as done:\n" + taskDescription;
+    }
+
+    /**
+     * Formats a message signaling a task has been successfully unmarked.
+     *
+     * @param taskDescription The description of the task unmarked.
+     */
+    public String getTaskUnmarkedMsg(String taskDescription) {
```

Comment:
Perhaps rename Msg to Message so it will be clearer for people who might not be familiar with short forms? 😸